### PR TITLE
Fixed issues with Customer Export

### PR DESCRIPTION
**Pull request by Shaik42**
I have fixed all the customer related issues and was able to export all the customers to Vikrant's local server.

**I have a query for the Customer Tickets:**
The login screen speaks about the "Maximum number of Tickets" and I think that if I enter some number there (say 100) then the application should fetch latest 100 tickets, but the logic written in RepairShopr application does this 
 -The entered number (here 100) will be compared with the Ticket's ID and only those Tickets whose ID are less than the entered number (100) will be exported. (It doesn't seem right).
So I need to confirm what is the meaning of "Maximum number of Tickets". 

Please find below all the changes in PR:
- Removed auto-generated files generated by Visual Studio.
- Removed Log files from bin as well.
- Corrected and skipped elements other than numbers and spaces in phone/mobile number of Customer contact information so that Customer creation does not fail for invalid numbers.
- If any contacts are missed due to Invalid Email IDs, they will be logged.
- Created new and included missing members to classes which were required to properly deserialize the JSON objects.
- Some exceptions were being ignored, but now those are also captured and logged into the log files
- Created a separate log file which shows only Errors (This log file will be found in the Roaming folder inside AppData).

### CODE REVIEW

**Comment by nasa42** [comment]
cc @theinventor

File: RepairShoprApps/app.config [hunk 1]
```diff
@@ -6,7 +6,8 @@
         </sectionGroup>
     </configSections>
    <appSettings>
-     <add key="HostName" value="repairshopr.com" />         
+     <!--<add key="HostName" value="repairshopr.com" />-->         
+    <add key="HostName" value="alfa.192.168.1.116.xip.io:3000" />
```
Comment (nasa42) [inline RepairShoprApps/app.config]: Please revert this to previous value.

**Comment by theinventor** [comment]
The "max tickets" thing was just a bad hack due to the bad algorithm used to get tickets. He wanted some kind of max number so the tool would know when to stop.

I would prefer we get rid of that and the tool just properly loop over every ticket.

@nasa42  and @Shaik42

**Comment by theinventor** [comment]
Can you confirm that the installer builder doesn't need any changes after this commit? Any new files, dlls, etc?

**Comment by theinventor** [comment]
Also confirm that contacts work, you may have to go to /settings/customers to enable sub contacts in repairshopr, then when there are multiple contacts on an "Account" in commitcrm, they should be created as "contacts" in RepairShopr. It may have already been working..

@nasa42

**Comment by nasa42** [comment]
Ok, I'll get that checked tomorrow

**Comment by Shaik42** [comment]
Hi Anderson,

No new files, dlls are added. Therefore the installer builder doesn't
require any change.

**Comment by Shaik42** [comment]
Sure, I will remove the textbox.

**Pull request merged**